VERSION-RANGE-SPEC.rst

```diff
@@ -642,10 +642,6 @@ These are a few known versioning schemes for some common Package URL
 
 These are generic schemes, to use sparingly for special cases:
 
-- **generic**: a generic version comparison algorithm (which will be specified
```

Comment:
there is no intention to remove this !

Reply:
Okay, then I will put it back in. Since it states it will be specified in the future, I was thinking it can be replaced now.